### PR TITLE
change loop from forEach to for/of

### PR DESCRIPTION
**Pull request by alexanderpaolini**
- Changed loops from forEach to for/of

### CODE REVIEW

**Comment by jpbberry** [comment]
why

**Comment by alexanderpaolini** [comment]
https://img.censor.bot/frog

**Comment by alexanderpaolini** [comment]
https://img.censor.bot/frog

**Comment by alexanderpaolini** [comment]
https://img.censor.bot/frog

**Comment by jpbberry** [comment]
frog

**Review by alexanderpaolini — COMMENTED** [review]
Great changes!

File: src/worker.js [hunk 1]
```diff
@@ -40,17 +40,17 @@ worker.commands
     }
   })
 
-sfw.forEach(type => {
```
Comment (jpbberry) [inline src/worker.js:43]: nah lmao

**Pull request closed without merging**